apps/console/src/features/groups/components/wizard/create-group-wizard.tsx

```diff
@@ -388,12 +401,12 @@ export const CreateGroupWizard: FunctionComponent<CreateGroupProps> = (props: Cr
     },{
         content: (
             viewRolePermissions
-                ? <RolePermissions
-                    data-testid={ `${ testId }-group-permission` }
+                ? (<RolePermissions
+                    data-testid={ `${testId}-group-permission` }
```

Comment:
```suggestion
                    data-testid={ `${ testId }-group-permission` }
```

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3153/commits/313c92a74b1c03a09f6d70e4b3f1f44530c6c246